JENKINS_HOME.gitignore

```diff
@@ -46,5 +46,11 @@ jobs/**/*workspace
 
 #!/secret*
 
+# Security warning: Following includes Jenkins' account settings
```

Comment:
The `s` is only omitted for possive when the object is plural, e.g., "users' rights".  Here, "Jenkins" is singular, thus `'s` is correct.
```suggestion
# Security warning: Following includes Jenkins's account settings
```

Reply:
@bl-ue
very thanks for comment. I learned so much.